process/merge_requests/README.md

```diff
@@ -39,27 +34,25 @@ These are the steps necessary:
 7. Press the blue "Compare branches" button.
 8. Enter title and description, following the guidelines written in the
    **Title and Description guidelines** section.  
-   Set no label, this will be done by someone else.  
-   Set no assignee, unless you've been asked to.
+9. Set label "Ready for review".  
```

Comment:
This is on a per project basis. The projects I mostly work in, for example, we consider all MRs as ready for review unless labeled as WIP.